### ISSUE TITLE
Add new call method to return supporting evms

### ISSUE THREAD

**Issue by sh-rn** [blocknetdx/exrproxy-env#180]
**Scenario:**
The XQuery (and future Hydra) dashboard app needs to display to the user which network(s) a particular SNode supports eg ETH, AVAX, SYS NEVM etc

**To do:**
- [ ] Write a containerised plugin for EXR env which can be deployed through the SNode builder, and which responds as an XRouter/XCloud Service to requests for a list of which SNodes support XQuery and/or Hydra, and which EVMs are supported by XQuery & Hydra on each of those SNodes.

**Comment by ConanMishler** [comment]
Yeah, that dashboard will be very user-friendly. Once this new PR is merged and SNodes start deploying with it, it should be possible to gather all the info displayed on that dashboard either by making api calls the SNode Explorer, or by gathering the same info directly from the Blocknet blockchain, like the SNode Explorer does.

Note, it is possible to configure an SNode to support different EVMs for Hydra than it does for XQuery (or visa versa). The above dashboard layout looks great, but if we use that format we'll want to give details under the "View details" button on which EVMs are supported by Hydra and which are supported by XQuery. (Maybe that was already the plan...)

**Comment by ConanMishler** [comment]
Now that https://github.com/blocknetdx/exrproxy-env/pull/185 has been merged, the task for this issue should really defined as:
> Write a containerized plugin for EXR env which can be deployed through the SNode builder, and which responds as an XRouter/XCloud Service to requests for a list of which SNodes support XQuery and/or Hydra, and which EVMs are supported by XQuery & Hydra on each of those SNodes.

**Comment by ConanMishler** [comment]
@shrnkld Now that @tryiou has added XRouter Service for `xrshowconfigs` to https://utils.blocknet.org (https://utils.blocknet.org/xrs/xrshowconfigs), it seems this issue can be closed, correct?

**Comment by shrnkld** [comment]
Would it still not be better to build this plugin and have every SNode deploy it so any dapps we build can fetch a list of available services directly from all available SNodes, instead of a single server/domain?

I wouldn't say this is urgent as utils.blocknet is sufficient for this XQuery dashboard MVP, but it seems a better solution assuming there are no network or other technical limitations to the XC plugin method.

**Comment by ConanMishler** [comment]
@shrnkld It might be simpler just to modify the SNode builder code to add the necessary config statements to `~/exrproxy-env/scripts/start-exrproxy.sh` so that http://<URL>/xrs/xrshowconfigs service is available on every SNode. Any reason not to do that? Maybe we should make that the task for this issue.

**Comment by shrnkld** [comment]
That sounds good to me @ConanMishler ⚡️
